### PR TITLE
Added officer policy page

### PR DESCRIPTION
**Pull request by BluuArc**
Work for #53. 

Accessible via Office Policy button on About page.

image

image

### CODE REVIEW

File: frontend-site/src/views/OfficePolicy.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    </v-layout>
+    <v-layout row>
+      <v-flex>
+        <v-card>
+          <v-card-title primary-title>
+            <h1 class="title">Space Policy</h1>
+          </v-card-title>
+          <v-card-text>
+            <span class="d-block">
+              The LUG office is a shared resource and thus, we have many people coming in any time during the day. While you are free to use the desks and chairs of the ACM, you may not 'hog' a seat. You must give up your seat to another person that wants to do any of the following:
```
Comment (Benbigmac) [inline frontend-site/src/views/OfficePolicy.vue]: good, just if on line 27 here you could change ACM to LUG. Sorry I missed that when I copy and pasted it

Comment (BluuArc) [reply frontend-site/src/views/OfficePolicy.vue]: Would the same apply to line 36 as well?

Comment (BluuArc) [reply frontend-site/src/views/OfficePolicy.vue]: Made changes to both lines. Let me know if you want me to change that second line back to ACM

Comment (Benbigmac) [reply frontend-site/src/views/OfficePolicy.vue]: yes both lines I didn't read it closely enough thanks for catching that

**Pull request merged**